Dockerfile

```diff
@@ -93,8 +91,8 @@ RUN pip install -e .
 FROM backend as prod
 
 # Install without a symlink
-RUN python setup.py sdist
-RUN pip install dist/ethyca-fides-*.tar.gz
+RUN python -m build --sdist
+RUN pip install dist/ethyca_fides-*\d+.\d+.\d+.*.tar.gz
```

Comment:
Hey @sanders41, just a side note here: According to [pep-0427](https://peps.python.org/pep-0427/#escaping-and-unicode), the naming convention for wheels should use underscores `_` rather than a dash `-`, because a dash is used to separate the name from the tag.

cc: @ofek